docker/install.sh

```diff
@@ -43,23 +32,24 @@ key="$1"
 case $key in
 
     -d|--driver)
-        DOCKER_DRIVER="$2"
+        if [[ "$2" != "" ]]; then
+            DOCKER_DRIVER="$2"
+        fi
```

Comment:
this is fine, just fyi, getopts can also be used- http://wiki.bash-hackers.org/howto/getopts_tutorial